docs/UserGuide.md

```diff
@@ -80,7 +80,9 @@ Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
 
 <box type="tip" seamless>
 
-**Tip:** A client can have any number of tags (including 0)
+**Tip 1:** Phone number can take on multiple formats, including for Singapore and foreign
+numbers (without the + symbol). This supports insurance agents with an international customer base. </br>
```

Comment:
realised that it dosen't support the +, so rather list it now to avoid it being considered a bug 😅

Reply:
ok